src/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
```

Comment:
before <add>, add <clear />

Reply:
this file is no longer required and removed.

---
In reply to: [633923991](https://github.com/microsoft/sarif-visualstudio-extension/pull/397#discussion_r633923991) [](ancestors = 633923991)